tools/RAiDER/gnss/processDelayFiles.py

```diff
@@ -0,0 +1,73 @@
+import datetime
+import glob
+import os
+
+import pandas as pd
```

Comment:
@jlmaurer  Could you ad in a brief run-example. Or add a a parser __main__ for now.

Reply:
Yes, I'll add that tonight after I get out of service!

Reply:
Ok do you want me to wait to merge it to have the example added?

Reply:
Sure that's fine, either way is fine with me.

Reply:
I'll add an rough example to the comments